README.md

```diff
@@ -40,7 +40,5 @@ See the file COPYING for more details.
 
 # The Build Process
 
-Refer to the file 'docs/src/code/building-linuxcnc.txt' for information
-about building and running the software, or look here:
```

Comment:
@kentavv  I'll add a link to http://linuxcnc.org/docs/2.9/html/code/building-linuxcnc.html as we have now a redirection from '2.9' to 'devel'